rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1373,6 +1373,11 @@ void main_window::OnEmuStop()
 	}
 	ui->actionManage_Users->setEnabled(true);
 
+	if (std::exchange(m_sys_menu_opened, false))
+	{
+		ui->sysSendOpenMenuAct->setText(tr("Send open system menu cmd"));
```

Comment:
Wrong

Reply:
Why? when m_sys_menu_opened is false the text should say "open".

Reply:
I meant something else. but I guess it doesn't matter.